exchange/websocket/connection.go

```diff
@@ -111,6 +113,7 @@ type Response struct {
 
 // connection contains all the data needed to send a message to a websocket connection
 type connection struct {
+	*subscription.Store
```

Comment:
🚧 Change request:
Not sure why we're embedding the pointer like this?
It seems wrong... Subscription store feels like it shouldn't intermingle with connection that openly.
```suggestion
	subscriptions *subscription.Store
```

Reply:
Yeah good catch, originally I wanted the methods exposed on the interface so we don't need to call the method to get the store but that was annoying.